cfme/utils/appliance/__init__.py

```diff
@@ -1543,6 +1543,52 @@ def update_rhel(self, *urls, **kwargs):
 
         return result
 
+    def upgrade(self, upgrade_to, cfme_only=True, reboot=False):
```

Comment:
This looks like redefinition of the method on L1334 in master?

https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/__init__.py